fuzzing/private/fuzz_test.bzl

```diff
@@ -41,28 +43,59 @@ def fuzzing_decoration(
         engine: The label of the fuzzing engine used to build the binary.
         corpus: A list of corpus files.
         dicts: A list of fuzzing dictionary files.
-        tags: Tags set on the fuzzing regression test.
+        instrument_binary: **(Experimental, may be removed in the future.)**
```

Comment:
if there is a plan, maybe explain what would be the behavior of the rule without this flag?

Reply:
Good point - I've clarified what the default behavior should be here. I'm going to assume that there is an implicit understanding that removing the flag will preserve the default behavior.